lib/DXIL/DxilModule.cpp

```diff
@@ -1826,6 +1827,56 @@ bool DxilModule::StripReflection() {
   return bChanged;
 }
 
+bool DxilModule::StripNamesSensitiveToDebug() {
+  bool changed = false;
+
+  if (!GetShaderModel()->IsLib()) {
+    // Strip struct names
+    unsigned nextStructId = 0;
```

Comment:
```suggestion
    unsigned NextStructId = 0;
```
nit: https://llvm.org/docs/CodingStandards.html#name-types-functions-variables-and-enumerators-properly.

There are some others lingering as well that would be nice to clean up.